lib/ansible/module_utils/rax.py

```diff
@@ -244,6 +244,7 @@ def rax_argument_spec():
     return dict(
         api_key=dict(type='str', aliases=['password'], no_log=True),
         auth_endpoint=dict(type='str'),
+        auth_token=dict(type='str'),
```

Comment:
This needs to be documented in the rackspace docs fragment.

Reply:
Thanks, I missed that. I've pushed the necessary changes (waiting for CI).

Reply:
CI passed.

Reply:
Since this is a credential `no_log=True` should be set here.